mfes/content/public/content/preview/coreplugins/org.sunbird.player.userswitcher-1.0/renderer/controller/sunbirdUserSwitcher.js

```
@@ -0,0 +1,286 @@
+                            } else {
+                                if (_.isUndefined($scope.TelemetryEvent.logGEEvent)) {
+                                    TelemetryService.interrupt("SWITCH", EkstepRendererAPI.getCurrentStageId()
+                                        ? EkstepRendererAPI.getCurrentStageId()
+                                        : $rootScope.pageId);
+                                }
+                                EkstepRendererAPI.dispatchEvent('renderer:telemetry:end');
+                                TelemetryService.setUser($rootScope.currentUser, EkstepRendererAPI.getCurrentStageId()
+                                    ? EkstepRendererAPI.getCurrentStageId()
+                                    : $rootScope.pageId, $scope.TelemetryEvent.interact);
+                                if (EkstepRendererAPI.getCurrentStageId()) { // If stage id is not available that means user is on Endpage.
+                                    var data = {};
```

Biome (1.9.4)

[error] Shouldn't redeclare 'data'. Consider to delete it or rename it.

'data' is defined here:


(lint/suspicious/noRedeclare)